src/client/Microsoft.Identity.Client/OAuth2/Throttling/HttpStatusProvider.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using Microsoft.Identity.Client.Internal.Requests;
+
+namespace Microsoft.Identity.Client.OAuth2.Throttling
+{
+    internal class HttpStatusProvider : IThrottlingProvider
+    {
+        /// <summary>
+        /// Default timespan that blocks an application, if HTTP 429 and HTTP 5xx was recieved and Retry-After HTTP header was not sent.
```

Comment:
> was not sent [](start = 122, length = 13)

nit: maybe call this ThrottlingCache as well, just to distinguish from tokenCache. just a thought. idk.